meta/saiserialize.c

```diff
@@ -813,8 +830,10 @@ int sai_deserialize_ip_address(
         _Out_ sai_ip_address_t *ip_address)
 {
     int res;
+    uint32_t idx;
+    const char *buf = buffer;
 
-    /* try first deserialize ip4 then ip6 */
+    /* try first deserialize ip4 then ip6 then uint8 array*/
```

Comment:
make no sense to try this as uint8, since you already know ip4 and ip6, no other types are supported

Reply:
Please see the comments for line 806